src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -68,20 +68,35 @@ class TransportManagementService implements Serializable {
 
         def proxy = config.proxy ? config.proxy : script.env.HTTP_PROXY
 
-        script.sh """#!/bin/sh -e
-                curl ${proxy ? '--proxy ' + proxy + ' ' : ''} -H 'Authorization: Bearer ${token}' -F 'file=@${file}' -F 'namedUser=${namedUser}' -o responseFileUpload.txt  --fail '${url}/v2/files/upload'
+        def responseFileUpload = 'responseFileUpload.txt'
+
+        def responseContent
+
+        try {
+            script.sh """#!/bin/sh -e
+                curl ${proxy ? '--proxy ' + proxy + ' ' : ''} -H 'Authorization: Bearer ${token}' -F 'file=@${file}' -F 'namedUser=${namedUser}' -o ${responseFileUpload}  --fail '${url}/v2/files/upload'
             """
 
-        def responseContent = script.readFile("responseFileUpload.txt")
+            echo("File upload successful.")
 
-        if (config.verbose) {
-            echo("${responseContent}")
-        }
+        } catch (Exception e) {
+            echo("Exception caught during file upload. Consider re-run in verbose mode in order to get more details.")
```

Comment:
[Picky]
```suggestion
            echo("Exception caught during file upload. Consider re-running in verbose mode in order to get more details.")
```
I dislike a little that this message is shown also if verbose is on. You don't want to check for `verbose` here again, right? Could be something like this:
```
def message = "Exception caught during file upload."
if (config.verbose) {
   message += " Consider re-run in verbose mode in order to get more details."
}
echo message
```

Reply:
... done